frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,148 @@
+    await operatorHubView.createSubscriptionFormLoaded();
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+      'OpenShift Serverless Operator',
+    );
+    await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+    await operatorHubView.allNamespacesInstallMode.click();
+
+    expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+    expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+    await operatorHubView.createSubscriptionFormBtn.click();
+    await crudView.isLoaded();
+    await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+    await crudView.isLoaded();
+    await catalogPageView.clickFilterCheckbox('installState-installed');
+    expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(
+      true,
+    );
+  });
+
+  it(`displays Operator in "Cluster Service Versions" view for "${testName}" namespace`, async () => {
+    await retry(() => catalogPageView.catalogTileFor('OpenShift Serverless Operator').click());
+    await browser
+      .wait(until.presenceOf(installServerlessOperator), 15000)
+      .then(() => browser.sleep(500));
+    await operatorHubView.viewInstalledOperator();
+    await crudView.isLoaded();
+    await browser.wait(
+      until.visibilityOf(operatorView.rowForOperator('OpenShift Serverless Operator')),
+    );
+    expect(operatorView.rowForOperator('OpenShift Serverless Operator').isDisplayed()).toBe(true);
+  });
+
+  it(`create knative-serving namespace and apply serving.yaml`, async () => {
+    await execSync(`oc new-project knative-serving`);
+    await execSync(
+      `oc apply -f ./packages/dev-console/integration-tests/views/serverless-service.yaml`,
+    );
+    await browser.get(`${appHost}/k8s/cluster/projects/`);
+    await crudView.isLoaded();
+    await browser.wait(until.elementToBeClickable(knativeServingNS));
+    expect(knativeServingNS.isDisplayed()).toBe(true);
+  });
+
+  it('Increase the Machine sets count', async () => {
+    await switchPerspective(Perspective.Administrator);
+    expect(sideHeader.getText()).toContain('Administrator');
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Compute')));
+    await sidenavView.clickNavLink(['Compute', 'Machine Sets']);
+    await crudView.isLoaded();
+
+    await browser.wait(until.elementToBeClickable(firstKebabMenu));
+    await firstKebabMenu.click();
+    await browser.wait(until.elementToBeClickable(crudView.actionForLabel('Edit Count')));
+    await crudView.actionForLabel('Edit Count').click();
+
+    await browser.wait(until.elementToBeClickable(machineSetsInput));
+    await machineSetsInput.sendKeys(protractor.Key.chord(protractor.Key.CONTROL, 'a'));
+    await machineSetsInput.sendKeys('6');
+    await browser.wait(until.elementToBeClickable(submitCount));
+    await submitCount.click();
+    expect(cellClass.isDisplayed()).toBe(true);
```

Comment:
here `cellClass` is `export const cellClass = element.all(by.css('[data-test-id="openshift-machine-api"]')).first();`

So if am not wrong you are just checking for first row to be present rather you should have checked if increase in machine sets is applicable.

Reply:
Increasing the machine sets takes a lot of time and it's not worth to stop the automation for this. And I have used the truthy statement before but as you asked me change I am just checking whether it is on the same page. I didn't find anything. I will try to find something but if I didn't find anything, we need to go with this.